robottelo/ui/base.py

```diff
@@ -57,7 +57,7 @@ def find_element(self, locator):
             return None
 
     def search_entity(self, element_name, element_locator, search_key=None,
-                      katello=None, timeout=None):
+                      katello=None, timeout=20):
```

Comment:
If this timeout is needed for SyncPlan, then I would suggest increasing this just for SyncPlan. On positive deletion tests we use search to check if the entity was really deleted and increasing the timeout from the base will increase the time UI automation need to complete.